core/server/data/migrations/utils.js

```diff
@@ -0,0 +1,244 @@
+// @ts-check
```

Comment:
We need to raise the topic of TS again I think :grimacing: All up for it, just need to know clear rules about if and how we introduce it into Ghost's codebase

Reply:
Ah yeah, I left this in, will remove it for now.